### PR TITLE
Feature/copy to

### PR DESCRIPTION
**Pull request by PodstDO**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -36,6 +36,21 @@ class Stack {
...
+        throw new Error(`${array} is not an Array!`);
+    }
+
+    if (index < 0) {
+        throw new Error('Index must be greater than 0!');
+    }
+
+    for (let i = 0; i < this.elements.length; i++) {
+        array[index] = this.elements[i];
+        index++;
```
Comment (Alexandra1003) [inline src/index.js]: you may use index +=1

File: src/index.js [hunk 2]
```diff
@@ -36,6 +36,21 @@ class Stack {
     return this.elements[this.elements.length - 1];
   }
 
+  copyTo(array, index) {
```
Comment (AndrZakhara) [inline src/index.js:39]: Your implementation dosen't work. Read documentation and fix method.
https://docs.microsoft.com/ru-ru/dotnet/api/system.collections.stack.copyto?view=netframework-4.7.2

**Review by AndrZakhara — CHANGES_REQUESTED** [review]
You must rewrite method

File: src/index.js [hunk 3]
```diff
@@ -36,6 +36,23 @@ class Stack {
...
+    if (!(array instanceof Array)) {
+      throw new Error(`${array} is not an Array!`);
+    }
+
+    if (index < 0) {
+      throw new Error('Index must be greater than 0!');
+    }
+
+    let elementUndex = index;
```
Comment (Alexandra1003) [inline src/index.js]: incorrect variable name

**Review by Alexandra1003 — APPROVED** [review]

**Review by AndrZakhara — APPROVED** [review]

**Pull request merged**